samples/cron-template.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         spec:
           containers:
           - name: create-vgs
-            image: registry.access.redhat.com/ubi8/ubi:8.4
+            image: registry.access.redhat.com/ubi8/ubi-minimal@sha256:e0814339ffc6c933652bed0c5f8b6416b9a3d40be2f49f95e6e4128387d2a24a
             imagePullPolicy: IfNotPresent
```

Comment:
can you add a comment that its 8.4 or 8.5 sha  ?

Reply:
Thanks Jai. thats what got me confused about whether it is 8.4 or 8.5 :)